services/src/kilda-core/kilda-model/src/main/java/org/openkilda/model/FlowPath.java

```diff
@@ -85,10 +85,18 @@
     @Convert(graphPropertyType = Long.class)
     private Cookie cookie;
 
+    @Property(name = "lldp_cookie")
+    @Convert(graphPropertyType = Long.class)
+    private Cookie lldpCookie;
```

Comment:
Did you consider introduce a separate entity for LLDP resource? E.g. LldpResources similar to EncapsulationResources.